interface/src/checksums_storage.rs

```diff
@@ -0,0 +1,149 @@
+use async_trait::async_trait;
+use solana_sdk::pubkey::Pubkey;
+
+/// Type of checksum for bubblegum epochs and account NFT buckets.
+/// It is technically a SHA3 hash.
+pub type Chksm = [u8; 32];
```

Comment:
I thought the type above was for this
```suggestion
    pub checksum: Option<Chksm>,
```

Reply:
```rust
pub type Chksm = [u8; 32];
```
is just an alias for a potential change of checksum type in the future (that will never happen)

Reply:
I believe it will just introduce a hell of wrap and unroll calls 🥲

Reply:
Deref for the rescue?

Reply:
Just thinking aloud tho, not a call to action